packages/core/include/zapierwrapper.mjs

```diff
@@ -0,0 +1,22 @@
+// not intended to be loaded via require() - copied during build step
+import { createRequire } from 'module';
+import path from 'node:path';
+import zapier from 'zapier-platform-core';
+
+const require = createRequire(import.meta.url);
+let appPath;
+
+try {
+    appPath = require.resolve(process.cwd());
+} catch (e) {
+    // if it failed, try to resolve the app path with self-referencing
+    // in case index.js is not in the root directory
+    try {
+        appPath = require.resolve(path.basename(process.cwd()));
```

Comment:
The package name should be taken from `package.json` rather than from the directory name.